src/presentation/http/router/join.test.ts

```diff
@@ -0,0 +1,74 @@
+import { describe, test, expect } from 'vitest';
+
+describe('Join API', () => {
+  describe('POST /join/:hash', () => {
+    test('Returns 406 when user is already in the team', async () => {
+      const expectedStatus = 406;
+      const hash = 'Hzh2hy4igf';
+      const userId = 2;
+      const accessToken = global.auth(userId);
+
+      const expectedResponse = {
+        message: 'Team with user specified user and note already exists',
+      };
+
+      const response = await global.api?.fakeRequest({
+        method: 'POST',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: `/join/${hash}`,
+      });
+
+      expect(response?.statusCode).toBe(expectedStatus);
+      expect(await response?.json()).toStrictEqual(expectedResponse);
```

Comment:
earlier we decided not to use `expectedStatus`, `expectedResponse` variables anymore

Reply:
removed